src/module_2/Module_2_EDA_Ex2.md

```diff
@@ -0,0 +1,1431 @@
+# Module 2: EDA
+
+## Exercise 2. Exploratory Data Analysis
+
+
+```python
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+from numpy import mean
```

Comment:
having imported numpy it is not cmmon to import then a concrete method. You can directly use np.mean